test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -34,7 +34,11 @@ SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 COMMON_SRCS="${SRC_DIR}/../common"
 HACK_DIR="${SRC_DIR}/../../../hack"
 
-"${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
+if [ -z "$USE_K3D" ]; then
```

Comment:
I would move that to  the setup-ks  as you in any case have different code for each platform there , so we can maintain the run-test agnostic to platform.

Reply:
Duplicating small routines like run-test.sh into multiple scripts differing by just one lines may not be the best answer.

Reply:
I agree , however multiple if(s)  are also problematic. Think of 5 different platforms :(